moveit_core/planning_scene/src/planning_scene.cpp

```diff
@@ -711,6 +711,18 @@ void PlanningScene::getPlanningSceneDiffMsg(moveit_msgs::PlanningScene& scene_ms
     if (do_omap)
       getOctomapMsg(scene_msg.world.octomap);
   }
+
+  // Ensure any detached collision objects get detached from the parent planning scene, too
```

Comment:
This comment is misleading: The parent planning scene is not (and should not get) modified.

Reply:
Agreed, I had a to-do to update the comment, but it got resolved somehow